index.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 const os = require('os');
 const fs = require('fs');
+const isDocker = require('is-docker');
 
 const isWsl = () => {
 	if (process.platform !== 'linux') {
 		return false;
 	}
 
```

Comment:
Wouldn't it be easier to just do the check here?

```js
if (isDocker()) {
	return false;
}
```

Reply:
I was avoiding this because I'm not sure what side-effects this can cause. To be safe I singled out my use case:
- first check if running in WSL;
- then check if running in docker.

I think checking for docker after the platform check will be fine, but I could not be sure for all use-cases.

Let me know if you if you if you want it changed.

Reply:
I agree